scopes/component/ui/component-size/component-size.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import fileSize from 'pretty-bytes';
+import { PillLabel } from '@teambit/design.ui.pill-label';
+import type { ComponentDescriptor } from '@teambit/component-descriptor';
+import { Tooltip } from '@teambit/design.ui.tooltip';
+import styles from './component-size.module.scss';
+
+export type ComponentSizeProps = { componentDescriptor: ComponentDescriptor } & React.HTMLAttributes<HTMLDivElement>;
+
+export function ComponentSize({ componentDescriptor, ...rest }: ComponentSizeProps) {
+  // TODO - fix type
+  const builderData: any = componentDescriptor.get('teambit.pipelines/builder');
+  // const builder = BuilderData.fromJson(a);
+  // TODO - find a better way to extract data and which type to use
+  const size = builderData?.aspectsData?.find(({ aspectId }) => aspectId === 'teambit.preview/preview')?.data?.size
+    ?.compressedTotal;
+  if (!size) return null;
+  return (
+    <Tooltip
+      className={styles.componentSizeTooltip}
+      placement="bottom"
+      content={
+        <div className={styles.componentSizeTooltipContent}>
+          Component bundle with dependencies, minified and gzipped
+        </div>
+      }
+    >
+      <div {...rest}>
+        <PillLabel>
+          <img style={{ width: '16px', marginRight: '4px' }} src="https://static.bit.dev/bit-icons/weight.svg" />
```

Comment:
stop using img svgs! either use icomoon, or make them into a react svg component

Reply:
ran doesnt want icomoon and I dont like adding it to the bundle. we can discuss this later and decide on a more general approach